docs/usage/filters/index.md

```diff
@@ -0,0 +1,89 @@
+---
+id: filters
+title: "Filters"
+sidebar_position: 3
+---
+
+# Filters
+
+Filters allows you to process incoming events and filter out the ones you don't want to see.
+To read how to develop your own filter, see the [Custom filter development](./development.md) guide.
+
+## Manage filters
+
+Botkube allows you to manage filters using **@Botkube** commands.
+
+### List available filters
+
+Run **@Botkube filters list** to get list of available filters and their running status:
+
+![List available filters](assets/filters_list.png)
+
+### Disable filter
+
+Run **@Botkube filters disable {filter-name}** to disable a filter named `{filter-name}`.
```

Comment:
in the kubectl usage we have `< .. >` and here `{ .. }`. I'm ok with both, but let's unify that.